.github/workflows/image-builder.yaml

```diff
@@ -37,9 +37,9 @@ jobs:
       shell: bash
       working-directory: "${{ github.workspace }}/add-outscale-image/images/capi"
       env:
-         OSC_ACCESS_KEY: ${{secrets.OSC_ACCESS_KEY}}
-         OSC_SECRET_KEY: ${{secrets.OSC_SECRET_KEY}}   
-         OUTSCALE_REGION: ${{secrets.OSC_REGION}}
+         OSC_ACCESS_KEY: ${{secrets.OSC_OPENSOURCE_ACCESS_KEY}}
+         OSC_SECRET_KEY: ${{secrets.OSC_OPENSOURCE_SECRET_KEY}}   
+         OUTSCALE_REGION: ${{secrets.OSC_OPENSOURCE_REGION}}
```

Comment:
why OUTSCALE_REGION or OSC_REGION

Reply:
https://www.packer.io/plugins/builders/outscale
In packer docs it is OUTSCALE_REGION .
we have to change to  OSC_REGION ?

Reply:
yes use OSC_REGION , the release of updated doc is not published yet